### DISCUSSION TITLE
polybar|error: Disabling module "tray" (reason: Unknown module: internal/tray)

### DISCUSSION THREAD

**Discussion by Memoframe** [polybar/polybar#2971] [Q&A] [answered]
Can't manage tray to work.
I have this in .config file:

`
modules-right = tray
`
and this 
`[module/tray]
type = internal/tray
tray-position = right`

I've read that earlier tray was not a module but now it is a built-in module.
Is it a thing in 3.6.3 or it's still not stable and I need to build?
If I need to build, I should uninstall 3.6.3?

Sry for newbish questions.

**Comment by patrick96** [comment, marked as answer]
The tray module is not in 3.6.3, you'll need to build from source.

> If I need to build, I should uninstall 3.6.3?

Yes

**Reply by Qurcaivel** [reply]
is there any documentation for tray module  customization?

**Reply by patrick96** [reply]
https://polybar.readthedocs.io/en/latest/user/modules/tray.html

This is for the latest unstable version

**Reply by Memoframe** [reply]
> https://polybar.readthedocs.io/en/latest/user/modules/tray.html
> 
> This is for the latest unstable version

So 3.6.3-206-g2cd0809a is not latest we can build?
Or it is latest and there is no access to 3.7.0 for now?
Or we have access but it's not recommended, especially for newbies?
Sorry for stupid questions, all this git thing is a bit overwhelming :D

**Reply by patrick96** [reply]
Version 3.7.0 does not exist, it is the version number of the next feature release. The documentation mentions it so that we don't have to go and add it everywhere once 3.7 is released, this way we can add the version information directly when documenting something.

Currently, `3.6.3-206-g2cd0809a` is the latest you can build. It contains changes that will be in 3.7, but are not yet in 3.6.3. In general, building these versions is only recommended if you are OK with stuff potentially breaking without notice.

It's good to ask these questions, besides the developers it may not be clear what this stuff means.
